runbotics/runbotics-orchestrator-ui/src/src-app/translations/en/process/process.json

```diff
@@ -326,6 +326,10 @@
     "Process.Details.Modeler.Actions.SharePointExcel.OpenFileFromRoot.FilePath.Info": "Refers to the location path, where the starting point is the folder, where the bot lands after login to the account, e.g. folderName/fileName.xlsx",
     "Process.Details.Modeler.Actions.SharePointExcel.OpenFileFromRoot.ListName.Info": "The default name is 'Documents'",
     "Process.Details.Modeler.Actions.CloudExcel.CloseSession.Label": "Close session",
+    "Process.Details.Modeler.Actions.CloudExcel.DeleteColumns.Label": "Delete columns",
+    "Process.Details.Modeler.Actions.CloudExcel.DeleteColumns.StartColumn": "Start column (value must be [A-Z])",
+    "Process.Details.Modeler.Actions.CloudExcel.DeleteColumns.EndColumn": "End column (value must be [A-Z])",
```

Comment:
In all excel desktop actions we have example value (and most of the restrictions) in the tooltip. 
+
[A-Z] contains regex expression which is not familiar to non-programmers. imo "uppercase letters A-Z" + example in the tooltip are enough